packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1616,9 +1616,35 @@ export default class Meeting extends StatelessWebexPlugin {
         VIDEO_TRACK: '1',
         SHARE_TRACK: '2'
       };
-      const trackMediaID = event.transceiver.mid;
       let eventType = null;
       const mediaTrack = event.track;
+      let trackMediaID = null;
+
+
+      // **In case of safari some time the transceiver is not present for specific os version
+      // sdk tries to determine the transceive using the track id present
+      if (event.transceiver) {
+        trackMediaID = event.transceiver.mid;
+      }
+      else {
+        const {audioTransceiver, videoTransceiver, shareTransceiver} = event.target;
+
+        // audio kind indicates its a audio stream
+        if (mediaTrack.id === audioTransceiver.receiver.track.id) {
+          trackMediaID = '0';
+        }
+        else
+        if (mediaTrack.id === videoTransceiver.receiver.track.id) {
+          trackMediaID = '1';
+        }
+        else
+        if (mediaTrack.id === shareTransceiver.receiver.track.id) {
+          trackMediaID = '2';
+        }
+        else {
```

Comment:
do we still need this?

Reply:
just a precaution , dont want it to break with no reason